code/game/objects/items/food/martian.dm

```diff
@@ -783,7 +783,7 @@
 		/datum/reagent/consumable/nutriment/vitamin = 2,
 	)
 	tastes = list("chicken" = 1, "coconut" = 1, "curry" = 1)
-	foodtypes = MEAT | VEGETABLES | DAIRY
+	foodtypes = MEAT | VEGETABLES | DAIRY //should the coconut milk count as dairy?
```

Comment:
If the dairy foodtype is counted as "this would upset someone lactose intolerant" then coconut shouldn't count for it


now does coconut count for fruit, then

Reply:
coconut should count for fruit

Reply:
Are coconuts a fruit or a nut, help.

Reply:
They are a drupe.

Reply:
> They are a drupe.

Almonds are also a drupe, but we classify it as a nut.

Reply:
with how fucked up food types are in general (they're used for both tastes *and* intolerances, blech), probably best just removing the dairy tag